presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayRemoveFunction.java

```diff
@@ -78,8 +78,42 @@ public Block remove(
             @SqlType("array(E)") Block array,
             @SqlType("E") Object value)
     {
-        BlockBuilder blockBuilder = type.createBlockBuilder(null, array.getPositionCount());
+        int found = -1;
```

Comment:
can we rename this to `valueIndex`

Reply:
I think this is better more descriptive. I can change it to foundIndex to indicate the intent of the logic.

Reply:
yeah index in variable name is more clear.